src/smolvm/runtime/_libkrun_launcher.py

```
@@ -0,0 +1,148 @@
+
+def main(argv: list[str]) -> int:
+    if len(argv) < 2:
+        print("usage: python -m smolvm.runtime._libkrun_launcher <config.json>", file=sys.stderr)
+        return 2
+
+    config = json.loads(Path(argv[1]).read_text())
+    ssh_host_port = int(config.get("ssh_host_port") or 2222)
+    kernel_format = int(config.get("kernel_format", KERNEL_FORMAT_RAW))
+
+    gvproxy_proc: subprocess.Popen | None = None
+    sock_path = f"/tmp/krun-gvproxy-{os.getpid()}.sock"
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/krun-gvproxy-"

(S108)